### PR TITLE
sql-query: multiple queries in the same spec

### PR DESCRIPTION
**Pull request by apahim**
Depends on https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/12421

### CODE REVIEW

**Review by maorfr — APPROVED** [review]

**Comment by maorfr** [comment]
thank you, as always, for the great commits!

**Review by rporres — APPROVED** [review]
I like the idea of multiple `psql|mysql` commands to be able to time every query separately. Good stuff! :)

File: reconcile/sql_query.py [hunk 1]
```diff
@@ -208,45 +217,64 @@ def collect_queries(query_name=None):
...
+
+def filesystem_closing_message():
     return [
-        '> /tmp/query-result.txt;',
         'echo;',
         'echo Get the sql-query results with:;',
         'echo;',
-        'echo  oc rsh --shell=/bin/bash ${HOSTNAME} ',
-        'cat /tmp/query-result.txt;',
+        'echo  oc cp ${HOSTNAME}:/tmp/query-result.txt ',
```
Comment (BumbleFeng) [inline reconcile/sql_query.py:276]: I get a warning when running this command and it looks like it fails
`tar: Removing leading '/' from member names`
Could we remove the leading '/'?

Comment (apahim) [reply reconcile/sql_query.py:276]: You tell me :)

**Pull request merged**